Specify sdl-config to use instead of using host sdl-config.

Signed-off-by: Koen Martens <gmc@sonologic.nl>

diff -Naur a/utils/Makefile b/utils/Makefile
--- a/utils/Makefile	2017-06-16 18:48:28.402824790 +0200
+++ b/utils/Makefile	2017-06-16 18:48:53.898696647 +0200
@@ -69,10 +69,10 @@
 	$(CC) $(CFLAGS) $(CPPFLAGS) -funsigned-char $^ $(LDFLAGS) -lm -o $@
 
 ffmvforce.o: ffmvforce.c
-	$(CC) $(CFLAGS) $(CPPFLAGS) -c $^ -o $@ `sdl-config --cflags`
+	$(CC) $(CFLAGS) $(CPPFLAGS) -c $^ -o $@ `${SDL_CONFIG} --cflags`
 
 ffmvforce: ffmvforce.o
-	$(CC) $^ -o $@ $(LDFLAGS) -g -lm `sdl-config --libs`
+	$(CC) $^ -o $@ $(LDFLAGS) -g -lm `${SDL_CONFIG} --libs`
 
 axbtnmap.o: axbtnmap.c axbtnmap.h
 
